cpp/examples/abm.cpp

```diff
@@ -380,13 +380,13 @@ void create_assign_locations(mio::abm::World& world)
  * Assign an infection state to each person.
  */
 
-void assign_infection_state(mio::abm::World& world, double exposed_pct, double infected_pct, double carrier_pct,
+void assign_infection_state(mio::abm::World& world, double exposed_pct, double infected_symptoms_pct, double infected_no_symptoms_pct,
```

Comment:
where did actually the `_pct` came from and what did it stand for?

Reply:
I assume this is "percentage", i. e., `infected_symptoms_pct = 0.7` for a 70% probability to be in that group.

Reply:
Ah okay. We should maybe think of changing this. Maybe "rel", "perc" or "prob" seem better understandble for me.

Reply:
Changed to "prob".